server/controller/index.js

```diff
@@ -57,3 +57,32 @@ exports.getId = async (ctx) => {
       return ctx.body;
     });
 };
+
+
+exports.markSpam = async (ctx) => {
+  ctx.user = { id: '5a2ff257fc13ae6d59000535' };
+  if (!ctx.user) {
+    ctx.body = 'Not authourized';
+    return ctx.body;
+  }
+  await Question.findOne({ _id: ctx.params.id})
+    .then(async (data) => {
```

Comment:
You don't need to include `async` here the function `markSpam` is already asynchronous.

Reply:
It should be here in order we could get a response from db.